docs/UserGuide.md

```diff
@@ -83,6 +87,7 @@ Adds a person to the address book and adds existing event to this person if even
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [evt/EVENT_INDEX]…​`
 
 * Person name cannot exceed 50 characters.
+* Person name is restricted to alphanumeric characters. 
```

Comment:
Would this be misleading because the user might expect an error message if they enter only numbers for the person's name

Reply:
It means we don't check how the user use this alphanumeric characters, as long as all the characters are from the set of alphanumeric characters, it is legal. Therefore, the user reading this should not expect an error if they key in all numbers because it is satisfies the condition of it using only alphanumeric characters.